compiler-rt/lib/hwasan/hwasan_report.cpp

```diff
@@ -221,29 +221,55 @@ static void PrintStackAllocations(const StackAllocationsRingBuffer *sa,
       for (LocalInfo &local : frame.locals) {
         if (!local.has_frame_offset || !local.has_size || !local.has_tag_offset)
           continue;
+        if (!(local.name && internal_strlen(local.name)) &&
+            !(local.function_name && internal_strlen(local.name)) &&
+            !local.decl_file)
```

Comment:
why don't we check this for zero size? please add in comment

Reply:
will check